content/geometry/human-index.adoc

```diff
@@ -0,0 +1,30 @@
+= Human structure
+:toc:
+:toclevels: 7
+
+//include::human-structure/Floor.adoc[leveloffset=+1] Removed as stated on the Miro board
```

Comment:
In Vehicle and Road we have the following structure:
- introduction
- general
- model-creation
- model-structure

Should we use this structure for the human class, too?